script/phase.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+#source ./config_lo.sh
+#source ./config_hg38.sh
+source ./config.sh
+
+#mkdir -p ${REF_PANEL_DIR}
```

Comment:
If we're going to have these bash scripts after all, best to remove commented lines.

Reply:
NA